neofetch

```diff
@@ -3805,23 +3820,43 @@ get_local_ip() {
     esac
 }
 
-get_public_ip() {
+get_public_ipv4() {
+    if type -p dig >/dev/null; then
+        public_ipv4="$(dig +time=1 +tries=1 +short myip.opendns.com @resolver1.opendns.com)"
```

Comment:
This is the same dig command as for IPv4. How will it return the IPv6 address in one call and the IPv4 address in anither? Same question for drill.

Also I would list curl and wget before dig and drill, as 
`dig +time=1 +tries=1 +short myip.opendns.com @resolver1.opendns.com`
fails for me in Windows/WSL

Also, `get_public_ipv4` and `get_public_ipv6` are have common logic. Perhaps move it to `_get_public_ip` and call it with params?

Reply:
They are pulling from different URLs. Right now while at home sometimes the URL will return IPv4 and sometimes it will return IPv6. So each time I run it I get a different IP address. The logic is the same because there is no need to change it except for the URL and by having 2 separate options it allows a user to choose either/or or both.

This is just what I found by using the default site of ident.me from the config. I just followed the logic based on a certain URL. I am open for other ideas also.